pytri/__init__.py

```diff
@@ -15,6 +15,9 @@
 limitations under the License.
 """
 
+from base64 import b64encode
+from io import BytesIO
+from PIL import Image
```

Comment:
I know this is usually bad form, but can we move this import to inside the `imshow` function call? (We should probably do the same thing with networkx)

That way users that are just doing scatters or somesuch don't need to install a c-bindings library